src/validate.ts

```diff
@@ -1,36 +1,20 @@
 'use strict';
 
-const Ajv = require('ajv'); 
-const ajv = new Ajv();
-const chai = require('chai');
 chai.use(require('chai-json-schema-ajv'));
 
-//calling the JSON schemas to throw assertion errors
-const sync_response_schema = require('../intents/sync.response.schema.json');
-const apiresponse = require('./apiresponse.json');
-
-
-var testsPassed = 0
-var testsFailed = 0
-var totalTests = 0
-
-/**
- * Generate requests to validate the responses later on  
-*/
-
 function generateRandom(min, max){
     //helper function for obtaining a random requestId
     return Math.floor(Math.random() * (max-min) + min).toString();
 }
 
 
 function generateSyncReq(){
-    const requestRandId = generateRandom(100,999);
+    const requestId = generateRandom(100,999);
     return {
-    "requestId": requestRandId,
-    "inputs": [{
-      "intent": "action.devices.SYNC"
-        }]
+        requestId,
+        "inputs": [{
+          "intent": action.devices.SYNC
```

Comment:
The keys do not need quotes, but the values do. Otherwise it should not build.

```javascript
return {
    requestId,
    inputs: [{
        intent: 'action.devices.SYNC'
    }]
}
```